src/components/Activity/index.tsx

```diff
@@ -6,28 +6,27 @@ interface ActivityProps {
   title: string;
   day: number;
   children?: React.ReactNode;
+  imageOrientation?: "left" | "right";
 }
 
 const Activity: FunctionComponent<ActivityProps> = ({
   logo,
   title,
   day,
   children,
+  imageOrientation,
 }) => {
+  imageOrientation = imageOrientation || "left";
   return (
-    <section className="flex w-full flex-col items-center justify-center text-center">
-      <div className="flex min-h-[16.5rem] flex-col items-center justify-center">
-        <div id="w-full">
-          <Image src={logo} alt={`Atividade ${title}`} className="w-32" />
-        </div>
-        <h5 className="my-5 text-2xl text-gray-600">
-          {title}
-          <br />
-          (Dia {day})
-        </h5>
-      </div>
+    <section className="grid w-full grind-rows-3 grid-cols-1 lg:grid-rows-2 lg:grid-cols-2 items-center justify-center place-contant-center">
+      <h5 className="mt-5 mb-24 ml-10 lg:col-span-2 font-good__times text-5xl text-current font-bold">
```

Comment:
Check responsiveness when the screen is very small. The margin-left is too big.

Also, remove from here the font-good__times too